components/Form/Checkbox/Checkbox.tsx

```diff
@@ -44,7 +51,7 @@ function Checkbox({ field: { name, value, ...field }, form: { errors }, id, labe
       </Label>
 
       <ErrorMessage name={name}>
-        {message => {
+        {(message: string) => {
```

Comment:
hmmm this shouldn't be needed. If you look in Formik's deps, `ErrorMessage` isn't typed?

Reply:
If I remove the `string` typing, I just get the TS flag. Not sure how else to get around that.

<img width="871" alt="Screenshot 2024-03-15 at 8 46 26 AM" src="https://github.com/OperationCode/front-end/assets/46408716/43b1721b-d407-481e-9fa0-978966854604">